packages/connect-explorer/src/js/data/methods/management/getFeatures.ts

```diff
@@ -0,0 +1,12 @@
+const name = 'getFeatures';
+const docs = 'methods/getFeatures.md';
+
+export default [
+    {
+        url: '/method/getFeatures',
+        name,
+        // docs,
```

Comment:
why is this commented?

Reply:
transfered from previous repo. I am going to check all these "fixtures" once again.

Reply:
removed. along with all other lines of unused / commented code I found 2de6c227b33358de5367219d2d71c1fefd6d9dfe and e27cd53a71677e02df824033244276f81b053445